modules/dashboard-samples/src/main/webapp/WEB-INF/deployments/showcaseWorkspace.workspace

```diff
@@ -66,6 +66,12 @@
     <param name="title" value="Apresentação de Painéis" lang="pt" />
     <param name="title" value="Demostración de cuadros de mando" lang="es" />
     <param name="title" value="Dashboards Schaufenster" lang="de" />
+    <param name="title" value="ダッシュボードショーケース" lang="ja" />
+    <param name="title" value="Демонстрационный пример" lang="ru" />
+    <param name="title" value="Démo de tableaux de bord" lang="fr" />
+    <param name="title" value="Apresentação de Painéis" lang="pt" />
+    <param name="title" value="Demostración de cuadros de mando" lang="es" />
+    <param name="title" value="Dashboards Schaufenster" lang="de" />

```

Comment:
Do we need really need this? It seems to just duplicate translations which are already in the file (and already duplicated dozens of times).